### DISCUSSION TITLE
Using Cheers through Chatterino

### DISCUSSION THREAD

**Discussion by Nexus8th** [Chatterino/chatterino2#3244] [Q&A] [answered]
Hi, is there any way to do that instead of going on Twitch? thanks

**Comment by Mm2PL** [comment, marked as answer]
Simply nope. I don't suspect twitch to add a public API to spend money on someone's behalf, that leads to all sorts of trouble.

**Reply by Nexus8th** [reply]
Thank you for the quick reply!
